src/bgw/job.c

```diff
@@ -965,8 +966,13 @@ ts_bgw_job_insert_relation(Name application_name, Name job_type, Interval *sched
 		values[AttrNumberGetAttrOffset(Anum_bgw_job_config)] = JsonbPGetDatum(config);
 
 	ts_catalog_database_info_become_owner(ts_catalog_database_info_get(), &sec_ctx);
-	values[AttrNumberGetAttrOffset(Anum_bgw_job_id)] =
-		ts_catalog_table_next_seq_id(catalog, BGW_JOB);
+
+	job_id = DatumGetInt32(ts_catalog_table_next_seq_id(catalog, BGW_JOB));
+	snprintf(app_name, NAMEDATALEN, "%s [%d]", NameStr(*application_name), job_id);
+
+	values[AttrNumberGetAttrOffset(Anum_bgw_job_id)] = Int32GetDatum(job_id);
+	values[AttrNumberGetAttrOffset(Anum_bgw_job_application_name)] = CStringGetDatum(app_name);
```

Comment:
this makes no sense, Anum_table_column constants are defined in catalog.h and match exactly the columns in the table so your suggested change will just make this not compile unless you want me to rename columns in the bgw_job table

Reply:
No, this was a suggestion for changing the name of the column since it is not really the application name any more, rather it is the job name (application name plus job id). I leave the decision if you think it is a good idea or not to you, it is merely an observation that it is no longer the application name and hence it might make sense to change the name of the column.